index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/70b9059057.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./index.css">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <header class="tool-bar">
+      <h2 class="logo">John Doe</h2>
+      <i class="fa-solid fa-bars fa-2x"></i>
+    </header>
+    <div class="headline">
+      <hgroup>
+        <h1 class="head-main-title">Hey There. I'm Alex</h1>
+        <h3 class="head-main-sub-title">I’am a Software Developer</h3>
+      </hgroup>
+      <p class="head-main-description">
+        I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+      </p>
+    </div>
+    <div class="social">
+      <a href=""><i class="fa-brands fa-github"></i></a>
+      <a href=""><i class="fa-brands fa-linkedin-in"></i></a>
+      <a href=""><i class="fa-regular fa-hand-peace"></i></a>
+      <a href=""><i class="fa-brands fa-twitter"></i></a>
+      <a href=""><i class="fa-solid fa-m"></i></a>
```

Comment:
Kindly consider putting those  `a` tags inside a `ul` tag, and each one of them inside a `li` tag because it makes your website more semantic